js/chips.js

```diff
@@ -66,6 +66,7 @@
       });
     };
 
+    // Handle events
     this.handleEvents = function(){
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) This function has 113 lines, which is greater than the 100 lines authorized. Split it into smaller functions. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://192.168.99.100:32770/coding_rules#rule_key=javascript%3AS138)
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Function has a complexity of 33 which is greater than 10 authorized. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://192.168.99.100:32770/coding_rules#rule_key=javascript%3AFunctionComplexity)